app/Pages/JoinGamePage.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { SafeAreaView, TextInput, View, Text, StyleSheet } from 'react-native';
+import GlobalStyles, { globalStyling } from '../styles/global';
+import EnterButton from '../component/EnterButton';
+
+const JoinGamePage = () => {
+  return (
+    <SafeAreaView style={[GlobalStyles.safeAreaView, Styles.joinGameContainer]}>
+      <Text style={[Styles.text, Styles.headerText]}>Guess Whom</Text>
+      <Text style={[Styles.text, Styles.subText]}>Please Enter Your Real Name</Text>
+      <TextInput style={Styles.inputBox}></TextInput>
+      <Text style={[Styles.text, Styles.subText]}> Please Enter Room</Text>
+      <TextInput style={Styles.inputBox}></TextInput>
+      <EnterButton />
```

Comment:
start putting these in a string file that just contains the text

Reply:
"please Enter room" etc...

Reply:
changes made